README.md

```diff
@@ -47,48 +76,37 @@ Article DOI: 10.xxxx/zenodo.xxxxx
 Article URL: https://zenodo.org/record/xxxxxx/file/article.pdf
 ```
 
-You must also assign the issue, volume, and article numbers. Have a
-look on the published articles and assign the next available number in
-the same issue and same volume if your article is still in the same
-year.  Otherwise, increment the volume number and start with
-issue 1. The issue number is only changed for accommodating special issues.
-Post your numbers to [this GitHub issue](https://github.com/ReScience/ReScience/issues/48)
-that serves to avoid that two editors assign the same numbers to two
-different articles.
+6\. If no errors were returned, you have successfully reserved a DOI! Next we’ll grab an issue, volume and article numbers.
+
+```
+NOTE: This DOI will not resolve anywhere. This behavior is expected
+```
 
+### Update the metadata
 
-#### 2. Metadata update and creation of the final PDF
+In this step, you'll update `metadata.yaml`, pull request the file back to the author, generate a new PDF (which will now contain the volume, issue, page, doi information), and copy that back here.
 
-The `metadata.yaml` has a section (at the end) for information added during
-the publication process. You should complete it and verify the whole file
-before moving on. The information you must add is:
+7\. Look at the last number on  [this GitHub issue](https://github.com/ReScience/ReScience/issues/48) and choose the next one in the series. Post a comment to claim that issue for your article. This comment
+serves to avoid that two editors assign the same numbers to two
+different articles.
+
+8\. Then add these two bits of information (volume and article number) along with the Zenodo DOI and URL to `metadata.yaml`. 
+
+You should complete the missing information and verify the whole file before moving on. The information you must add are:
   - DOI (from Zenodo)
   - article URL (from Zenodo)
   - contributors (reviewers and editors), with ORCIDs
   - acceptance and publication date
   - issue, volume, and article numbers.
 
-Next, there are two situations:
-
- - If the authors have used the [ReScience article template](http://github.com/rescience/template),
-   you can fork their repository, copy the edited `metadata.yaml` to it, and create
-   the final PDF by running `make`. Don't forget to submit a pull request to the
-   authors' repository with the changes you made to `metadata.yaml`.
-
- - Otherwise, you give the authors the required information and
-   ask them to update their article and produce the final PDF file. You should
-   check that their PDF contains all the information you sent them.
+9\.  Pull request just the `metadata.yaml` back to the author's repo (this will mean copying this file back to the author repo fork). Once the pull-request is merged, have them `Make` a new `article.pdf`. The PDF will now contain the volume, article number and DOI.
```

Comment:
have them `Make` -> ask them to prepare

If the authors don't use our template, there's a good chance they are not using LaTeX either, nor probably `make`.